docs/pages/versions/unversioned/sdk/sqlite.mdx

```diff
@@ -8,49 +8,64 @@ import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import PlatformsSection from '~/components/plugins/PlatformsSection';
 import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
 
 `expo-sqlite` gives your app access to a database that can be queried through a [WebSQL](https://www.w3.org/TR/webdatabase/)-like API. The database is persisted across restarts of your app.
 
-An [example to do list app](https://github.com/expo/examples/tree/master/with-sqlite) is available that uses this module for storage.
-
 <PlatformsSection android emulator ios simulator />
 
-## Guides
+## Installation
+
+<APIInstallSection />
+
+## Usage
+
+An [example to-do list app](https://github.com/expo/examples/tree/master/with-sqlite) is available that uses this module for storage.
```

Comment:
Can we use BoxLink here, like we plan to do for other examples?

<img width="1615" alt="Screenshot 2022-11-23 at 13 23 04" src="https://user-images.githubusercontent.com/719641/204238649-4d7c0e83-4991-42bd-bf34-3d7607ce37d8.png">

Reply:
I was thinking of using it before opening the PR 😄

Reply:
Done!